respa_exchange/listener.py

```diff
@@ -283,7 +313,8 @@ def handle_events(self):
         changed_resources = set()
         while True:
             try:
-                event = self.events.get(block=False)
+                event = self.events.get(timeout=1)
+                self.events.task_done()
```

Comment:
`task_done` seems to be new in Python 3.4.4. It might be worth it to check for its existence before calling it...